pages/index.jsx

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+import Head from "next/head";
+import { NavBar } from "../src/components/Nav/NavBar";
+import { useState } from "react";
+import ReactPlayer from "react-player";
+import { LogoImage } from "../src/components/Header/Logo";
+
+export const Home = () => {
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  return (
+    <>
+      <Head>
+        <title>Corporate Campus Configurator</title>
+      </Head>
+      <LogoImage />
+      <NavBar />
+      <h1>CORPORATE CAMPUS CONFIGURATOR</h1>
+
+      <StyledVideo>
+        <>
+          <ReactPlayer
+            url="https://www.youtube.com/watch?v=xJL-s9dq7V0"
+            width="100%"
+            height="180px"
+            controls={true}
```

Comment:
Tip: there is a shorthand for setting boolean values as props:
```suggestion
            controls
```
Ofc this only works if you want to set it to true. False would then be set by the absence of `controls`.